packages/fuzzer/coverage.ts

```diff
@@ -17,54 +17,54 @@
 import { addon } from "./addon";
 
 export class CoverageTracker {
-	private static readonly MAX_NUM_COUNTERS: number = 1 << 20;
-	private static readonly INITIAL_NUM_COUNTERS: number = 1 << 9;
-	private readonly coverageMap: Buffer;
-	private currentNumCounters: number;
+    private static readonly MAX_NUM_COUNTERS: number = 1048576; // 1 << 20
+    private static readonly INITIAL_NUM_COUNTERS: number = 512; // 1 << 9
+    private readonly coverageMap: number[]; // Using an array instead of a Buffer
```

Comment:
We use Buffer for performance reasons---it's just so much faster than arrays, and takes 8 times less memory than an array of numbers. Try writing some bytes (counter values are bytes) a billion times to a buffer and the same with an array of numbers and see how long it takes in both cases.
This part of the code should be as fast as possible, because it will be executed many times in every fuzzer iteratrion.